USERS.md

```diff
@@ -50,3 +50,4 @@ Organizations below are **officially** using Argo Events. Please send a PR with
 1. [Yubo](https://www.yubo.live/)
 1. [WooliesX](https://wooliesx.com.au/)
 1. [Woolworths Group](https://www.woolworthsgroup.com.au/)
+1. [Enso Security](https://enso.security)
```

Comment:
Can you update it with alphabetical order?

Reply:
Sure )